### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by vanshitalavange**
Created grid layouts:
- two-column layout
- three-column layout
- autofill grid layout
image

### CODE REVIEW

File: stylesheets/components/grid.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.grid-wrapper {
+  display: grid;
+}
+.two-col-layout {
+  grid-template-columns: 1fr 1fr;
+}
+.three-col-layout {
+  grid-template-columns: 1fr 1fr 1fr;
```
Comment (sadathshariff) [inline stylesheets/components/grid.css]: Just a suggestion to use repeat(3,1fr) rest the code looks good and good naming convention

**Review by sadathshariff — COMMENTED** [review]
Looks good have given a suggestion do look into that

File: stylesheets/components/grid.css (hunk 1 above)
Comment (vanshitalavange) [reply stylesheets/components/grid.css]: thank you for the suggestion! I've done the changes

**Review by sadathshariff — APPROVED** [review]

**Review by Gautam2305 — COMMENTED** [review]
looks good to merge

**Pull request merged**